package.json

```diff
@@ -11,6 +11,7 @@
     "build": "npm run clean && node_modules/webpack/bin/webpack.js",
     "codecov": "cat ./coverage/coverage-final.json | ./node_modules/codecov.io/bin/codecov.io.js",
     "clean": "rm -rf ./static/assets/*",
+    "install-git": "npm install git+https://git@github.com/18F/cg-style.git#master && cd node_modules/cloudgov-style && npm install && npm run build; cd ../../",
```

Comment:
@msecret are you wedded to calling this script "install-git"? It is a bit confusing, I think something like "cloudgov-style" would be a bit more obvious what this script is doing.

In addition, do we want to add this command into the docs or do we want this to run in something like `scripts:postinstall` so other devs don't need to remember to run this command?